readme.md

```diff
@@ -233,6 +234,7 @@ dockerRun {
 - `volumes` optional map of volumes to mount in the container. The key is path
   to the host volume, relative to the project folder, the value is the exposed
   container volume path.
+- `ports` optional map of local port to container port.
```

Comment:
it's not a map, at least not in the same sense as the `volumes` flag is a map.

Reply:
can you just change this to: 
```
optional mapping `local:container` of local port to container port.
```

Reply:
Will do.